### ISSUE TITLE
/usr/share/hashcat/OpenCL/m-0004_a0-pure.cl: No such file or directory

### ISSUE THREAD

**Issue by Tobey123** [hashcat/hashcat#3400]
run hashcat in kali and mac
but all show the information below
/usr/share/hashcat/OpenCL/m-0004_a0-pure.cl: No such file or directory

But I can't find the file "m-0004_a0-pure.cl"

Thanks.

**Comment by Chick3nman** [comment]
You've got a corrupt module or custom module that is broken. Download a fresh copy of hashcat and try again.

**Comment by philsmd** [comment]
What is the full command that you use (all switches / arguments / parameters) ?

I agree with @Chick3nman that it seems to be either a problem of your installation/setup or how you use it. I can't reproduce this (new ?) problem ? Does other/older hashcat versions from https://hashcat.net/hashcat work for you ?

**Comment by Tobey123** [comment]
hashcat -a 0 -m 16500 a.hash 1.dict
I‘d try downloading a fresh copy of hashcat and try again.
But still the same.

**Comment by philsmd** [comment]
if you have a local copy, you need to run it with a prefix of "./" e.g. something like `./hashcat --help`

maybe try to clean up a little bit older versions or installed versions (that's probably the reason for the `/usr/share/hashcat/` path).

**Comment by Tobey123** [comment]
Yes, I have a local copy, I run it with a prefix of "./" 
I try to install the new clean os and fresh copy of hashcat and try again.
But still same........Orz.........

**Comment by philsmd** [comment]
Does it still say something about `/usr/share/hashcat/` ?

do you use your operating system package manager to install it ?

if you run it locally, it should not say anything about `/usr/share/hashcat`.

What distro do you use ? Please give more details. Thx

**Comment by NAs3c** [comment]
Hello,

Having the same problem, trying to launch hashcat on Kali VM

```
┌──(kali㉿kali)-[~/redacted]
└─$ ~/Tools/hashcat-6.2.6/hashcat.bin -a 0 -m 16500 jwt.hash trivial.lst -r /usr/share/hashcat/rules/* 
hashcat (v6.2.6) starting

OpenCL API (OpenCL 3.0 PoCL 3.1+debian  Linux, None+Asserts, RELOC, SPIR, LLVM 15.0.6, SLEEF, DISTRO, POCL_DEBUG) - Platform #1 [The pocl project]
==================================================================================================================================================
* Device #1: pthread-sandybridge-11th Gen Intel(R) Core(TM) i5-1135G7 @ 2.40GHz, 1436/2936 MB (512 MB allocatable), 2MCU

...

Watchdog: Temperature abort trigger set to 90c

/home/kali/Tools/hashcat-6.2.6/OpenCL/m-0004_a0-pure.cl: No such file or directory

Started: Tue Jun  6 03:59:24 2023
Stopped: Tue Jun  6 03:59:34 2023
                                                                                                                                         
┌──(kali㉿kali)-[~/redacted]
└─$ uname -a            
Linux kali 6.1.0-kali9-amd64 #1 SMP PREEMPT_DYNAMIC Debian 6.1.27-1kali1 (2023-05-12) x86_64 GNU/Linux
```

Any solution has been found ?

**Comment by NAs3c** [comment]
Ok, after triple checking, the problem was coming from the JWT token in `jwt.hash`. With a well formed token, everything works fine.

Problem solved for me,

**Comment by matrix** [comment]
> Ok, after triple checking, the problem was coming from the JWT token in `jwt.hash`. With a well formed token, everything works fine.
> 
> Problem solved for me,

Hi @NAs3c, could you share jwt.hash? Thanks

**Comment by NAs3c** [comment]
Hello @matrix,

I cannot give the original jwt, but here's one that can reproduce the bug : 

`eyJuYW1lIjoic2F2ZV9lbWFpbCIsImFsZyI6IkhTMjU2IiwiZ2xvYmFsIjp0cnVlfQ.eyJwYXJhbWV0ZXJzIjpbeyJwYXJhbTEiOiJ2YWx1ZTEifSx7InBhcmFtMiI6InZhbHVlMiJ9XX0.9Lgg6s7X5ReZcH784ldvcC0SLjTouXz4XhmxfX9UqOg==`

=> Just add the trailing '==' to a well formed jwt that doesn't need it

**Comment by matrix** [comment]
fixed by #3760
